cmd/amazon-cloudwatch-agent/register_event_logger_notwindows.go

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+// +build !windows
+
+package main
+
+func RegisterEventLogger() error {
+	// Unreachable code, do nothing
+	return nil
```

Comment:
It would help me understand this change if there was a comment explaining why this doesn't do anything on non-Windows. Is this because of the lumberjack dependency?

Reply:
Will add some explanatory comment for this.